raw/GB/language-tables/Central_Veps.csv

```diff
@@ -1,196 +1,196 @@
+"GB296","Is there a phonologically or morphosyntactically definable class of ideophones that includes ideophones depicting imagery beyond sound?",0,"Grünthal 2015:in passim",,,,,
+"GB297","Can polar interrogation be indicated by a V-not-V construction?",0,"Grünthal 2015:224–231",,,,,
+"GB298","Can standard negation be marked by an inflecting word (‘auxiliary verb’)?",1,"Grünthal 2015: 127",,"GB298",,,
+"GB299","Can standard negation be marked by a non-inflecting word (‘auxiliary particle’)?",0,"Grünthal 2015:127",,,,,
+"GB300","Does the verb for ‘give’ have suppletive verb forms?",0,"Grünthal 2015:in passim",,,,,
+"GB301","Is there an inclusory construction?",1,"Grünthal 2015: 289",,,,,
+"GB302","Is there a phonologically free passive marker (‘particle’ or ‘auxiliary’)?",0,"Grünthal 2015:250",,,,,
+"GB303","Is there a phonologically free antipassive marker (‘particle’ or ‘auxiliary’)?",0,"Grünthal 2015:in passim",,,,,
+"GB304","Can the agent be expressed overtly in a passive clause?",0,"Grünthal 2015:121–126",,,,,
+"GB305","Is there a phonologically independent reflexive marker?",1,"Grünthal 2015: 224",,"GB305",,,
+"GB306","Is there a phonologically independent reciprocal marker that does not consist of two quantifiers (e.g. each + other",0,"Grünthal 2015:in passim",,,,,
+"GB309","Are there multiple past or multiple future tenses"," distinguishing distance from Time of Reference?",0,"Grünthal 2015:128",259,,,
+"GB312","Is there overt morphological marking on the verb dedicated to mood?",1,"Grünthal 2015: 141",,"GB312",,,
+"GB313","Are there special adnominal possessive pronouns that are not formed by an otherwise regular process?",0,"Grünthal 2015:105",,,,,
+"GB314","Can augmentative meaning be expressed productively by a shift of noun class/gender?",0,"Grünthal 2015:in passim",,,,,
+"GB315","Can diminutive meaning be expressed productively by a shift of noun class/gender?",0,"Grünthal 2015:in passim",,,,,
+"GB316","Is singular number regularly marked in the noun phrase by a phonologically free element?",0,"Grünthal 2015:in passim",,,,,
+"GB317","Is dual number regularly marked in the noun phrase by a phonologically free element?",0,"Grünthal 2015:in passim",,,,,
+"GB318","Is plural number regularly marked in the noun phrase by a phonologically free element?",0,"Grünthal 2015:in passim",,,,,
+"GB319","Is trial number regularly marked in the noun phrase by a phonologically free element?",0,"Grünthal 2015:in passim",,,,,
+"GB320","Is paucal number regularly marked in the noun phrase by a phonologically free element?",0,"Grünthal 2015:in passim",,,,,
+"GB321","Is there a large class of nouns whose noun class/gender is not phonologically or semantically predictable?",0,"Grünthal 2015:in passim",,,,,
+"GB322","Is there grammatical marking of direct evidence (perceived with the senses)?",0,"Grünthal 2015:in passim",,,,,
+"GB323","Is there grammatical marking of indirect evidence (hearsay"," inference"," etc.)?",0,"Grünthal 2015:in passim",,,
```

Comment:
That's not really a solution. Now you quoted all the columns, but **after** they already have been split on inner commas. Below is what csvstat reports for the Value column:
```
  3. "Value"

	Type of data:          Text
	Contains null values:  False
	Unique values:         16
	Longest value:         77, characters
	Most common values:    0 (125x)
	                       1 (51x)
	                        A or P)? (4x)
	                       3 (2x)
	                        A or P indexes (do include indexes that also mark TAM or negation)? (2x)
```

Reply:
Darnit, I expeclitly went back and used an earlier semi-colon separated version of the file and then quoted all fields. I'll try again.

Reply:
Now what I've done in 1ad7a144a8073c502e72907a56bfa4dc4cbfec3a is take the file Central_veps as it existed at 1ad7a144a8073c502e72907a56bfa4dc4cbfec3a .

Reply:
Now "Kettunen & Siro 1935" is missing from line 187. I am trying to find out why that is.

Reply:
I'm not going to lie, the commit history is not pretty. However, I believe I have now solved the problem The separator is `,` and all fields are quoted and the `Kettunen & Siro`  source is correct. I revert to the  state at https://github.com/HedvigS/uratyp-1/commit/62d14182234c1810571a8a6e34c640a3a5298f3c and then did the sep and source changes.